vault/files/vault_systemd.service.jinja

```diff
@@ -7,6 +7,11 @@ After=network-online.target consul.service
 [Service]
 EnvironmentFile=-/etc/sysconfig/vault
 Restart=on-failure
-ExecStart=/usr/local/bin/vault server {% if vault.dev_mode %}-dev{% else %} -config="/etc/vault/config/server.hcl"{% endif %}
+ExecStart=/usr/local/bin/vault server {% if vault.dev_mode %}-dev{% else %} -config="/etc/vault/config/server.json"{% endif %}
 User={{ vault.user }}
 Group={{ vault.group }}
+{% if grains['os'] in ['CentOS', 'RedHat'] -%}
```

Comment:
im not sure this will work with  docker/travis

Reply:
I'll double check for that. I know I had to fiddle with that on `kitchen-salt`.